### PR TITLE
pg: make query logging configurable

### PR DESCRIPTION
**Pull request by daemonfire300**

### CODE REVIEW

File: backend/postgres/postgres.go [hunk 1]
```diff
@@ -228,15 +232,50 @@ func CustomConnectionPool(opts *pg.Options) *pg.DB {
...
+
+const (
+	readMode  queryMode = iota
+	writeMode queryMode = iota
+)
+
+// determineQueryMode is a poorman's attempt at checking whether the query is a read or write to the database.
+// Feel free to improve.
+func determineQueryMode(qry string) queryMode {
+	if strings.HasPrefix(strings.ToLower(qry), "select") {
```
Comment (daemonfire300) [inline backend/postgres/postgres.go:257]: could probably benefit from a LeftTrim too

File: backend/postgres/postgres.go [hunk 2]
```diff
@@ -228,15 +232,50 @@ func CustomConnectionPool(opts *pg.Options) *pg.DB {
 		Str("as", opts.ApplicationName).
 		Msg("PostgreSQL connection pool created")
 	db := pg.Connect(opts)
-	db.OnQueryProcessed(queryLogger)
+	if cfg.LogWrite || cfg.LogRead {
+		db.OnQueryProcessed(queryLogger)
+	} else {
+		log.Logger().Warn().Msg("Connection pool has logging queries disabled completely")
```
Comment (jpwenzel) [inline backend/postgres/postgres.go:238]: Is this an information that we would want to log on `Info` level, assuming it happens only once per opened connection?

File: backend/postgres/postgres.go (hunk 1 above)
Comment (jpwenzel) [reply backend/postgres/postgres.go:257]: Use `TrimSpace` maybe?

Comment (daemonfire300) [reply backend/postgres/postgres.go:257]: Done

**Review by pieceofsoul — APPROVED** [review]

**Pull request merged**